src/util.cpp

```diff
@@ -943,18 +952,38 @@ void ArgsManager::ReadConfigStream(std::istream& stream)
     }
 }
 
-void ArgsManager::ReadConfigFile(const std::string& confPath)
+void ArgsManager::ReadConfigFiles()
 {
     {
         LOCK(cs_args);
         m_config_args.clear();
     }
 
+    const std::string confPath = GetArg("-conf", BITCOIN_CONF_FILENAME);
     fs::ifstream stream(GetConfigFile(confPath));
 
     // ok to not have a config file
     if (stream.good()) {
         ReadConfigStream(stream);
+        std::vector<std::string> includeconf(GetArgs("-includeconf"));
+        {
+            // We haven't set m_network yet (that happens in SelectParams()), so manually check
+            // for network.includeconf args.
+            std::vector<std::string> includeconf_net(GetArgs(std::string("-") + GetChainName() + ".includeconf"));
+            includeconf.insert(includeconf.end(), includeconf_net.begin(), includeconf_net.end());
+        }
+
+        if (includeconf.size() > 0) {
```

Comment:
Remove this condition. If it's empty then there is nothing to loop below.

Reply:
If someone does `-noincludeconf`, I believe it will exist, but the vector will be empty. See line 682, which adds the key to the map, but clears it (the vector, not the map).

Reply:
Sorry, line 950, not 682. 682 is for `m_override_args`.

Reply:
Actually, nevermind, you're right. The for loop will just skip.